src/toil/jobStores/aws/jobStore.py

```diff
@@ -120,7 +120,7 @@ def copyPart(partIndex):
             with attempt:
                 dstBucket = s3.get_bucket(dstBucketName)
                 upload = dstBucket.initiate_multipart_upload(dstKeyName, headers=headers)
-        log.info("Initiated multipart copy from 's3://%s/%s' to 's3://%s/%s'.",
+        log.debug("Initiated multipart copy from 's3://%s/%s' to 's3://%s/%s'.",
```

Comment:
These I would leave INFO because otherwise there is no sign of anything happening during the import of large files. @tetron asked for this, IIRC.